### PR TITLE
0.2019-05-27: added man pages

### PR DESCRIPTION
**Pull request by zvezdochiot**
Added man pages

### CODE REVIEW

File: man/man1/pdfsizeopt.1 [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+\fB--do-debug-gs=\fRYES_NO
+default: \fIno\fR
+  Display debug info about where pdfsizeopt is trying to find Ghostscript, whether the found Ghostscripts work, and which one was chosen? All this before reading the first input PDF.
+
+.SH EXAMPLE
+\fBpdfsizeopt\fR sample.pdf
+
+.SH COPYRIGHT
+Copyright 2007 Péter Szabó.
+ All rights reserved.
```
Comment (pts) [inline man/man1/pdfsizeopt.1]: Please add a comment stating that the software is licensed under GPL-2.0.

File: man/man1/pdfsizeopt.1 [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+\fBpdfsizeopt\fR
+
+.SH DESCRIPTION
+PdfSizeOpt - PDF file size optimizer.
+
+.SH SYNOPSIS
+\fBpdfsizeopt\fR [<flag>...] <input.pdf> [<output.pdf>]
+
+.SH OPTIONS
+.TP
```
Comment (pts) [inline man/man1/pdfsizeopt.1]: Please remove the descriptions of options. Even though they are correct now, they may become obsolete over time, and pdfsizeopt doesn't have enough contributor time to keep it up-to-date. Please add something like this instead: See pdfsizept --help for options.

File: man/man1/pdfsizeopt.1 [hunk 3]
```diff
@@ -0,0 +1,185 @@
+.TH "PdfSizeOpt" "1" "27 May 2019" "0.2019-05-27" "User Manual"
```
Comment (pts) [inline man/man1/pdfsizeopt.1]: PdfSizeOpt  -->  pdfsizeopt

File: man/man1/pdfsizeopt.1 [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+.SH EXAMPLE
+\fBpdfsizeopt\fR sample.pdf
+
+.SH COPYRIGHT
+Copyright 2007 Péter Szabó.
+ All rights reserved.
+
+.SH SEE ALSO
+ pdfinfo(1), pngout(1), jpdftweak(1).
```
Comment (pts) [inline man/man1/pdfsizeopt.1]: jpdftweak is not relevant, please remove it. Please add jbig2(1), sam2p(1), imgdataopt(1). gs(1) and qpdf(1).

File: man/man1/pdfsizeopt.1 [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+.SH CONTACTS
+ Autors: Péter Szabó (https://github.com/pts)
```
Comment (pts) [inline man/man1/pdfsizeopt.1]: Autors  -->  Authors

**Comment by zvezdochiot** [comment]
@pts say> Please remove the descriptions of options.

I pass.

**Pull request closed without merging**